backend/infrahub/workflows/models.py

```diff
@@ -51,15 +51,18 @@ def full_name(self) -> str:
 
     def to_deployment(self) -> dict[str, Any]:
         payload: dict[str, Any] = {"name": self.name, "entrypoint": self.entrypoint, "tags": self.get_tags()}
-        if self.type == WorkflowType.INTERNAL:
+        if self.type == WorkflowType.CORE:
```

Comment:
Should `INTERNAL` workflows also have `version` filled, as they had before?

Reply:
the version isn't really used today, I'll take care of that in another PR